user_input/user_input.go

```diff
@@ -0,0 +1,42 @@
+package user_input
+
+import (
+	"fmt"
+	"strings"
+)
+
+func GetLetter() string {
+	var letter string
+	fmt.Print("Type a letter: ")
+	fmt.Scan(&letter)
+	fmt.Println("Your letter is:", letter)
+	return letter
+
+}
+
+func GuessLetter(word string, guessedChars []string) string {
```

Comment:
As a general rule, a function shoud do one thing and do it well. This keeps code easy to change and maintainable. It also makes it easier to track down bugs. 


This function does a lot, it..
1. request user input 
2. checks that input against `guessedChars`
3. checks the input against the actual `word`
4. prints the result of those checks.

These pieces of functionality should be refactored into different functions